test/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ option( MIOPEN_TEST_GFX908 "Test arch gfx908 mode" OFF )
 option(MIOPEN_TEST_CONV Off)
 option(MIOPEN_TEST_DEEPBENCH Off)
 option(MIOPEN_TEST_DRIVER_ITER_MODE Off)
+option( MIOPEN_TEST_MIOTENSILE "Test MIOpenTensile path" OFF )
```

Comment:
[Question] Shall we automatically set it to "On" when `MIOPEN_USE_MIOPENTENSILE=On`?

Reply:
I think we want them OFF as default. This mode is dedicated for GEMM related tests with miopentensile.

Reply:
Then in which cases and how we will enable these tests?

Reply:
Right, looks like tests are disabled. Enabled in new commit.